doc/classes/ResourceImporterWAV.xml

```diff
@@ -23,11 +23,12 @@
 			The end loop point to use when [member edit/loop_mode] is [b]Forward[/b], [b]Ping-Pong[/b], or [b]Backward[/b]. This is set in samples after the beginning of the audio file. A value of [code]-1[/code] uses the end of the audio file as the end loop point.
 		</member>
 		<member name="edit/loop_mode" type="int" setter="" getter="" default="0">
-			Controls how audio should loop. This is automatically read from the WAV metadata on import.
-			[b]Disabled:[/b] Don't loop audio, even if metadata indicates the file should be played back looping.
-			[b]Forward:[/b] Standard audio looping.
-			[b]Ping-Pong:[/b] Play audio forward until it's done playing, then play it backward and repeat. This is similar to mirrored texture repeat, but for audio.
-			[b]Backward:[/b] Play audio in reverse and loop back to the end when done playing.
+			Controls how audio should loop.
+			- [b]Detect From WAV:[/b] Uses loop information from the WAV metadata.
+			- [b]Disabled:[/b] Don't loop audio, even if the metadata indicates the file playback should loop.
+			- [b]Forward:[/b] Standard audio looping. Plays the audio forward from the begin point to the end point, then loops back.
+			- [b]Ping-Pong:[/b] Plays the audio forward until the end point, then backwards to the begin point, repeating this cycle.
+			- [b]Backward:[/b] Plays the audio backwards from the end point to the begin point, then loops back.
```

Comment:
General thoughts:
- Instead of saying "begin point" and "end point" you could directly reference the corresponding properties (`[member edit/loop_begin]` and `[member edit/loop_end]`). Not sure how readable that would be in practice. 
    - Alternatively, I don't see the harm in saying "beginning" and "end" here, but I'm not too much of an audio expert.
- It would be nice if the descriptions of Forward, Ping-Poing, and Backward all ended the same for consistency. That is, "then loops back".
    - That would not sound nice, so maybe it's proper to say "then repeats this cycle" or similar.

Reply:
I think just "then repeats" is simpler and easier to understand.